recipes/erlang/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "18.3" %}
+
+package:
+  name: erlang
+  version: {{ version }}
+
+source:
+  fn: otp_src_{{ version }}.tar.gz
+  url: http://erlang.org/download/otp_src_{{ version }}.tar.gz
+  sha256: "fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3"
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - m4
+    - perl
+    - readline 6.2*
+    - openssl 1.0.2g
```

Comment:
So, based on the [OpenSSL FAQ]( https://www.openssl.org/docs/faq.html#MISC8 ) I think I agree that this is too restrictive. We definitely should not be pinning the letter. We may not even need to pin the minor number (3rd) as ABI is preserved across that.